src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodIgnoreThrows.java

```diff
@@ -75,12 +75,12 @@ private static void catchAndRethrowSame(Object o) {
      */
     private static void catchAndRethrowDifferent(Object o, int i) {
         try {
-            float x = 1 / i; // ArithmeticException when i = 0
+            float x = 1 / i;                // ArithmeticException when i = 0
         } catch (RuntimeException ex) {
```

Comment:
Please review all lines, there should be no extra spaces

Reply:
Done.